sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -515,15 +537,27 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 		}
 
 		if extSrc == nil {
+			entityType := networkgraph.InternetEntity()
+			isExternal, err := conn.IsExternal()
+			if err != nil {
+				// IP is malformed or unknown - assume it is external and log a warning
```

Comment:
Self-correction:
```suggestion
				// IP is malformed or unknown - do not show on the graph and log the info 
```

Reply:
Fixed in [b99179c](https://github.com/stackrox/stackrox/pull/9718/commits/b99179c12d93ca4ae401de930506af0e6a2eaca9)